### PR TITLE
further link light admin pages

### PR DESCRIPTION
**Pull request by mtbc**
Beyond those from #1714 adds a couple more links among light admin pages. See https://trello.com/c/kDxd1MeO/57-cross-reference-light-admin-docs for background.

### CODE REVIEW

File: omero/sysadmins/admins-with-restricted-privileges.rst [hunk 1]
```diff
@@ -17,9 +17,11 @@ facility managers, image analysts, or anybody who needs to organize
 users and data of others in OMERO.
 
 Full administrators in OMERO can create new administrators with
-restricted privileges using the OMERO.web
-interface, see the :help:`create new users <sharing-data#admin>` section
-of our Help documentation.
+restricted privileges using the OMERO.web interface, see the
+:help:`create new users <sharing-data#admin>` section of our Help
+documentation. With considerably more difficulty one may also :doc:`use
```
Comment (hflynn) [inline omero/sysadmins/admins-with-restricted-privileges.rst]: Do we want to rephrase this to say it is possible but not recommended? Maybe:
"Support for creating restricted administrators is yet to be added to OMERO.cli. It is possible to [use OMERO.cli to adjust restrictions] on an existing administrator but we do not recommend this as the [permissions mapping] may cause confusion."

**Comment by hflynn** [comment]
I realise it's not strictly within the scope of this PR but I also find the title of the permissions mapping page somewhat difficult to parse and also wonder whether that page would be better NOT listed in the main index and instead added to an invisible TOC at the bottom of the main light admins page.

**Comment by mtbc** [comment]
Invisible TOC sounds scary but if my suggested retitling does not suffice then I'll happily accept a PR. :smiley:

**Comment by hflynn** [comment]
Your retitling is indeed helpful, we can probably leave it there (someone else will probably complain if we hide the page ;))

File: omero/sysadmins/admins-with-restricted-privileges.rst [hunk 2]
```diff
@@ -17,9 +17,13 @@ facility managers, image analysts, or anybody who needs to organize
...
 
 Full administrators in OMERO can create new administrators with
-restricted privileges using the OMERO.web
-interface, see the :help:`create new users <sharing-data#admin>` section
-of our Help documentation.
+restricted privileges using the OMERO.web interface, see the
+:help:`create new users <sharing-data#admin>` section of our Help
+documentation. OMERO.cli does not yet support easy management of
+restrictions nor does it offer the helpful :doc:`permissions mapping
+<mapping-restricted-admins>` so we do not recommend :doc:`using
```
Comment (joshmoore) [inline omero/sysadmins/admins-with-restricted-privileges.rst]: Because there could be a mis-configuration or because it's more complicated? If the latter, perhaps this could be phrased as "advanced use".

File: omero/sysadmins/cli/light-admins.rst [hunk 3]
```diff
@@ -16,11 +16,13 @@ administrator restrictions in the awkward manner described hereunder.
...
-  checking *one* box in the web interface may lift *multiple* underlying
-  restrictions from an administrator. The recommended OMERO.web
-  management interface may thus prove confusing if OMERO.cli has been
-  used to set a combination of restrictions that does not correspond to
-  those bundles of related restrictions available in OMERO.web.
+  the :doc:`permissions mapping <../mapping-restricted-admins>` is such
+  that checking *one* box in the web interface may lift *multiple*
+  underlying restrictions from an administrator. The recommended
+  OMERO.web management interface may thus prove confusing if OMERO.cli
+  has been used to set a combination of restrictions that does not
```
Comment (joshmoore) [inline omero/sysadmins/cli/light-admins.rst:23]: This certainly worries me. Is this something that's slated for 5.4.1? and/or should we consider a CLI plugin that we can rollout more quickly?

**Review by joshmoore — COMMENTED** [review]
The specific changes read fine though I'm worried about the tone we're setting. It sounds somewhat undone, which is perhaps what you're trying to get across but it'd be good to be clear on the next steps.

File: omero/sysadmins/cli/light-admins.rst (hunk 3 above)
Comment (mtbc) [reply omero/sysadmins/cli/light-admins.rst:23]: Noted on https://trello.com/c/iZeXX5FD/17-manage-admin-privileges-from-cli though does require someone to more than just tweak existing CLI code.

**Comment by hflynn** [comment]
Having looked at what else we are doing on the index page, I'm thinking we should move the mapping page down a level in the TOC but I can do that on a new PR. Are you happy with this now @joshmoore ?

**Comment by joshmoore** [comment]
Thanks, @mtbc
